src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -77,10 +77,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     String user = userService.getCurrentUser().getEmail();
-    String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
+    String text = Jsoup.clean(request.getParameter("text"), Whitelist.basic());
     String recipient = request.getParameter("recipient");
 
-    Message message = new Message(user, MessageUtil.formatImages(text), recipient);
```

Comment:
Would we ever want a message that hasn't been formatted?

I'm wondering if it would work better to include the calls to format everything in the Message class (maybe in the constructor for Message), rather than calling them separately every time a Message is created.

Reply:
I made all the changes.

Reply:
Do I merge now?